rpc/test/helpers.go

```diff
@@ -180,70 +178,34 @@ func NewTendermint(app abci.Application, opts *Options) *nm.Node {
 		panic(err)
 	}
 
-	config.IPFS = cfg.DefaultIPFSConfig()
-	err = initIpfs(config, opts.loadIpfsPlugins, logger)
-	if err != nil {
-		panic(err)
-	}
 	node, err := nm.NewNode(config, pv, nodeKey, papp,
 		nm.DefaultGenesisDocProviderFunc(config),
 		nm.DefaultDBProvider,
+		mockIPFSAPIProvider,
```

Comment:
Shouldn't we either call the provider directly in the constructor (not OnStart) or even do this before and just pass in a ready CoreAPI.

Reply:
We can, can't tell if we should